Resources/Prototypes/Body/Organs/human.yml

```diff
@@ -7,6 +7,7 @@
     sprite: Mobs/Species/Human/organs.rsi
   - type: Organ
   - type: Food
+    quality: Nasty # Frontier
```

Comment:
Need to remove this to the food system PR.
Just uncomment it here and we will restore it later.